src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -8,21 +8,31 @@
 import com.revrobotics.CANSparkMax;
 import edu.wpi.first.util.datalog.DoubleLogEntry;
 import edu.wpi.first.wpilibj.DataLogManager;
+import edu.wpi.first.util.datalog.DataLog;
+import edu.wpi.first.util.datalog.DoubleLogEntry;
+import edu.wpi.first.wpilibj.DataLogManager;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.Commands;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants;
+import edu.wpi.first.wpilibj.PWM;
+import edu.wpi.first.wpilibj.PWM;
 
 public class Intake extends SubsystemBase {
   private final CANSparkMax m_motor;
   private final DoubleLogEntry log_output =
       new DoubleLogEntry(DataLogManager.getLog(), "/intake/output");
 
+  PWM lEDPwm = new PWM(0);
+
```

Comment:
make this final and fix naming
```suggestion
  private final PWM m_led = new PWM(0);
```